run.sh

```diff
@@ -1,69 +1,94 @@
 #!/usr/bin/env bash
 BASE_DIR=$(cd $(dirname $0); pwd -P)
-CASE_FILE=$2
-#export RUST_LOG=debug
+CASE_FILE="cases.json"
+MODE=""
 
-run_boring_server() {
-  cargo run -- \
-  --client $BASE_DIR/../dist/Debug/bin/nss_bogo_shim \
-  --server $BASE_DIR/../boringssl/build/ssl/test/bssl_shim \
-  --rootdir $BASE_DIR/../boringssl/ssl/test/runner/ \
-  --test-cases $BASE_DIR/$CASE_FILE \
-  --client-writes-first
+print_help() {
+  printf "Help: \n \
+        -v) Verbose output.\n \
+        -m) Test mode. [ all | loopback | ossl_server | ossl_client | bssl_server | bssl_shim ]\n \
+        -c) Case file. (Optional. Default cases.json) \n"
 }
 
-run_boring_client() {
-  cargo run -- \
-  --client $BASE_DIR/../boringssl/build/ssl/test/bssl_shim \
-  --server $BASE_DIR/../dist/Debug/bin/nss_bogo_shim \
-  --rootdir $BASE_DIR/../boringssl/ssl/test/runner/ \
-  --test-cases $BASE_DIR/$CASE_FILE
-}
+while [ $# -gt 0 ]; do
+    case $1 in
```

Comment:
Always use double quotes for variables unless you are certain that their contents won't cause problems.